packages/relay/src/lib/eth.ts

```diff
@@ -1054,7 +1054,10 @@ export class EthImpl implements Eth {
       }
 
       const contractCallResponse = await this.mirrorNodeClient.postContractCall(callData, requestId);
-      return contractCallResponse && contractCallResponse.result ? EthImpl.prepend0x(contractCallResponse.result) : EthImpl.emptyHex;
+      if (contractCallResponse && contractCallResponse.result) {
+        return contractCallResponse.result == EthImpl.emptyHex ? await this.callConsensusNode(call, gas, requestId) : EthImpl.prepend0x(contractCallResponse.result)
```

Comment:
Make it cleaner with a check for `EthImpl.emptyHex` that throws a `MirrorNodeClientError`.

is there a way to determine the type of case by the call being make so that not all `0x`'s are retried on?

Reply:
Also add a trace log to say this issue occured and we're retrying with consensus node so we can track it

Reply:
I get that we can't tell from the Mirror Node side now. So I think just the extra log and we're good.

Reply:
Actually @georgi-l95 I take that back we should be able to check the byte code value by calling `api/v1/contracts/(call.to}` and checking the runtime bytecode.
That should let us know we're in this specific scenario and retry.

I'm trying to avoid retrying on all `0x` responses cause many could be valid

Reply:
Yeah, i've added this check. Now if runtime_bytecode is 0x, we throw error and proceed with consensus.